src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -1243,10 +1243,16 @@ int RadosBucket::commit_logging_object(const std::string& obj_name,
     const std::string& prefix,
     std::string* last_committed) {
   rgw_pool data_pool;
+  rgw_pool head_data_pool;
   const rgw_obj head_obj{get_key(), obj_name};
   const auto placement_rule = get_placement_rule();
+  bool same_pool = false;
 
-  if (!store->getRados()->get_obj_data_pool(placement_rule, head_obj, &data_pool)) {
+  rgw_placement_rule tail_placement_rule;
+  tail_placement_rule.storage_class = "BUCKET_LOGGING";
```

Comment:
it is ok to have that hardcoded for now, but please pass it through the APIs from the source bucket conf.
later on we may want to add the storage class as a parameter to the bucket logging conf at the source

Reply:
Ack.